starter_code/bin/www

```diff
@@ -40,7 +40,7 @@ server.on('error', error => {
       throw error;
   }
 });
-
-server.listen(process.env.PORT, () => {
+//revisar esto
+server.listen(process.env.PORT ||3000, () => {
```

Comment:
This is actually a good practice, because normally when you deploy your app, the server will assign you a PORT but if you're working in local, the app will be running in the PORT=3000 :grin: